samples/chaincode/cc-tools-demo/README.md

```diff
@@ -0,0 +1,274 @@
+We provide a small shell script to make this task a bit easier.
+
+```bash
+export CC_ID=cc-tools-demo
+export CC_PATH="$FPC_PATH/samples/chaincode/cc-tools-demo/"
+export CC_VER=$(cat "$FPC_PATH/samples/chaincode/cc-tools-demo/mrenclave")
+cd $FPC_PATH/samples/deployment/test-network
+./installFPC.sh
+```
+
+Note that the `installFPC.sh` script returns an export statement you need to copy and paste in the terminal.
+This sets environment variables for the package IDs for each chaincode container.
+Continue by running:
+
+```bash
+make ercc-ecc-start
+```
+
+You should see now four containers running (i.e., `cc-tools-demo.peer0.org1`, `cc-tools-demo.peer0.org2`, `ercc.peer0.org1`, and `ercc.peer0.org2`).
+
+### Invoke simple asset
+
+Open a new terminal and connect to the `fpc-development-go-support` container by running
+
+```bash
+docker exec -it fpc-development-go-support /bin/bash
+```
+
+```bash
+# prepare connections profile
+cd $FPC_PATH/samples/deployment/test-network
+./update-connection.sh
+
+# # update the connection profile for external clients outside the fpc dev environment
+cd $FPC_PATH/samples/deployment/test-network
+./update-external-connection.sh
+
+# make fpcclient
+cd $FPC_PATH/samples/application/simple-cli-go
+make
+
+# export fpcclient settings
+export CC_NAME=cc-tools-demo
+export CHANNEL_NAME=mychannel
+export CORE_PEER_ADDRESS=localhost:7051
+export CORE_PEER_ID=peer0.org1.example.com
+export CORE_PEER_LOCALMSPID=Org1MSP
+export CORE_PEER_MSPCONFIGPATH=$FPC_PATH/samples/deployment/test-network/fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp
+export CORE_PEER_TLS_CERT_FILE=$FPC_PATH/samples/deployment/test-network/fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/server.crt
+export CORE_PEER_TLS_ENABLED="true"
+export CORE_PEER_TLS_KEY_FILE=$FPC_PATH/samples/deployment/test-network/fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/server.key
+export CORE_PEER_TLS_ROOTCERT_FILE=$FPC_PATH/samples/deployment/test-network/fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
+export ORDERER_CA=$FPC_PATH/samples/deployment/test-network/fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+export GATEWAY_CONFIG=$FPC_PATH/samples/deployment/test-network/fabric-samples/test-network/organizations/peerOrganizations/org1.example.com/connection-org1.yaml
+export FPC_ENABLED=true
+export RUN_CCAAS=true
+
+# init our enclave
+./fpcclient init $CORE_PEER_ID
+
+# interact with the FPC-CC-Tools Chaincode. The getSchema is a built-in cc-tools transaction
+./fpcclient invoke getSchema
+
+```
+
+Congratulations! You have successfully created a FPC chaincode with go and invoked it using our simple cli.
```

Comment:
The congratulations should also mention cc-tools ...

Reply:
Done